axelor-account/src/main/resources/i18n/messages_fr.csv

```diff
@@ -2339,6 +2354,7 @@
 "Partial Sale","Cession partielle",,
 "Partial Scrapping","Mise au rebus partielle",,
 "Partial cession","Cession Partielle",,
+"Partially validate selected reconcile groups","Valider partiellement le letrage sélectionné",,

```

Comment:
It must be "Temporarily reconcile selected lines" and translation must be "Lettrer temporairement les lignes sélectionnées"